<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReweightingTab</class>
 <widget class="QWidget" name="ReweightingTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>727</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditInputPMF">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelTemperature">
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTemperature">
       <property name="text">
        <string>300.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUnit">
       <property name="text">
        <string>Unit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxUnit">
       <item>
        <property name="text">
         <string>kcal/mol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kJ/mol</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxConvertToPMF">
       <property name="text">
        <string>Convert to PMF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBoxTrajectory">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Trajectory</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonRemoveTrajectory">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QListView" name="listViewTrajectory">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonAddTrajectory">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBoxReweightingAxis">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Reweighting Axis</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableView" name="tableViewReweightingAxis">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QPushButton" name="pushButtonRun">
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditOutput"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonHelp">
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelOutput">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonSaveTo">
     <property name="text">
      <string>Save to</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelFromColumns">
       <property name="text">
        <string>From column(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFromColumns"/>
     </item>
     <item>
      <widget class="QLabel" name="labelToColumns">
       <property name="text">
        <string>To column(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditToColumns"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelInputPMF">
     <property name="text">
      <string>Input PMF</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonOpen">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPushButton" name="pushButtonReadAxes">
     <property name="text">
      <string>Read axes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditInputPMF</tabstop>
  <tabstop>pushButtonOpen</tabstop>
  <tabstop>lineEditOutput</tabstop>
  <tabstop>pushButtonSaveTo</tabstop>
  <tabstop>lineEditFromColumns</tabstop>
  <tabstop>lineEditToColumns</tabstop>
  <tabstop>pushButtonHelp</tabstop>
  <tabstop>lineEditTemperature</tabstop>
  <tabstop>comboBoxUnit</tabstop>
  <tabstop>checkBoxConvertToPMF</tabstop>
  <tabstop>listViewTrajectory</tabstop>
  <tabstop>pushButtonAddTrajectory</tabstop>
  <tabstop>pushButtonRemoveTrajectory</tabstop>
  <tabstop>tableViewReweightingAxis</tabstop>
  <tabstop>pushButtonReadAxes</tabstop>
  <tabstop>pushButtonRun</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
